### ISSUE TITLE
[bilibili] Cannot extract formats higher than 480P

### ISSUE THREAD

**Issue by qianbinbin** [yt-dlp/yt-dlp#7581]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

China

### Provide a description that is worded well enough to be understood

Even cookies are provided using `--cookies-from-browser`, yt-dlp cannot extract formats > 480P at present, but I believe it's working a few days ago.

Watching online with Chrome is okay.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', '--cookies-from-browser=chrome', 'https://www.bilibili.com/video/BV1mX4y147LG', '-F']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (pip)
[debug] Python 3.11.4 (CPython x86_64 64bit) - macOS-10.15.7-x86_64-i386-64bit (OpenSSL 3.1.1 30 May 2023)
[debug] exe versions: ffmpeg 6.0 (setts), ffprobe 6.0, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[Cookies] Extracting cookies from chrome
[debug] Extracting cookies from: "/Users/qianbinbin/Library/Application Support/Google/Chrome/Default/Cookies"
[debug] using find-generic-password to obtain password from OSX keychain
[Cookies] Extracted 2437 cookies from chrome
[debug] cookie version breakdown: {'v10': 2410, 'other': 0, 'unencrypted': 27}
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
yt-dlp is up to date (stable@2023.07.06)
[BiliBili] Extracting URL: https://www.bilibili.com/video/BV1mX4y147LG
[BiliBili] 1mX4y147LG: Downloading webpage
[BiliBili] BV1mX4y147LG: Extracting videos in anthology
[BiliBili] Format(s) 1080P 高清, 720P 高清, 1080P 60帧 are missing; you have to login or become premium member to download them. Use --cookies-from-browser or --cookies for the authentication. See  https://github.com/yt-dlp/yt-dlp/wiki/FAQ#how-do-i-pass-cookies-to-yt-dlp  for how to manually pass cookies
[BiliBili] 357460305: Extracting chapters
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[info] Available formats for BV1mX4y147LG:
ID EXT RESOLUTION FPS │ FILESIZE  TBR PROTO │ VCODEC        VBR ACODEC      ABR
───────────────────────────────────────────────────────────────────────────────
0  m4a audio only     │ ≈2.47MiB  42k https │ audio only        mp4a.40.5   42k
1  m4a audio only     │ ≈5.67MiB  96k https │ audio only        mp4a.40.2   96k
2  m4a audio only     │ ≈9.42MiB 160k https │ audio only        mp4a.40.2  160k
3  mp4 640x360     29 │ ≈3.19MiB  54k https │ avc1.64001E   54k video only
4  mp4 640x360     30 │ ≈2.59MiB  44k https │ av01.0.08M.08 44k video only
5  mp4 640x360     30 │ ≈3.59MiB  61k https │ hev1.1.6.L120 61k video only
6  mp4 852x480     29 │ ≈5.20MiB  88k https │ avc1.64001F   88k video only
7  mp4 852x480     30 │ ≈3.32MiB  57k https │ av01.0.08M.08 57k video only
8  mp4 852x480     30 │ ≈4.85MiB  83k https │ hev1.1.6.L120 83k video only
```

**Comment by gamer191** [comment]
Do they show if you pass `--allow-unplayable-formats`?

**Comment by qianbinbin** [comment]
> Do they show if you pass `--allow-unplayable-formats`?

I've just tried but no luck.

Bilibili uses HEVC/AVC/AV1 for high bitrates videos, which are commonly supported.

This script works fine: https://github.com/the1812/Bilibili-Evolved

**Comment by bashonly** [comment]
Related: #5874, #5655

**Comment by qianbinbin** [comment]
Surprisingly I logged in with **Safari** and tried with `--cookies-from-browser=safari`, and **it works**.

After that I refreshed **Chrome** page and my account was logged out automatically. Then I logged in again, and `--cookies-from-browser=chrome` magically **works**.

Indeed Bilibili supports signing in on multiple browsers/devices with the same account. I guess it may be something wrong with the cookies, but for some reason Bilibili didn't tell me that.

Anyway, many thanks for your help! I'll close the issue.
